Source/ACE.Server/Network/SessionConnectionData.cs

```diff
@@ -8,7 +8,7 @@ public class SessionConnectionData
     {
         public UIntSequence PacketSequence { get; set; }
         public uint FragmentSequence { get; set; }
-        public ISAAC IssacClient { get; }
+        public ISAAC IssacClient { get; set; }
```

Comment:
will do this when i get home:
```suggestion
#if NETDIAG
        public ISAAC IssacClient { get; set; }
#else
        public ISAAC IssacClient { get; }
#endif
```